packages/plugin-core/src/commands/GotoNote.ts

```diff
@@ -80,8 +57,19 @@ type FoundLinkSelection = NonNullable<
 /**
  * Open or create a note. See {@link GotoNoteCommand.execute} for details
  */
-export class GotoNoteCommand extends BasicCommand<CommandOpts, CommandOutput> {
+export class GotoNoteCommand extends BasicCommand<
+  GoToNoteCommandOpts,
+  GoToNoteCommandOutput
+> {
   key = DENDRON_COMMANDS.GOTO_NOTE.key;
+  private extension: IDendronExtension;
```

Comment:
not required for this PR but does it make sense to make extension and wsUtils part of the command interface?

Reply:
By command interface do you mean adding IDendronExtension to BaseCommand?